src/_experimental_/GraphQL/QlHooks.re

```diff
@@ -167,6 +167,23 @@ module LoadPatron = [%graphql
   |}
 ];
 
+module LoadPatronNew = [%graphql
```

Comment:
Let's put a comment above here saying:
`// we currently have two patron objects while the graph is in a half updated state. When the graph is refactored these 'patron' queries will be merged into one.